ciao-controller/openstack_image.go

```diff
@@ -185,7 +188,7 @@ type Config struct {
 	Password string
 }
 
-func getIdentityClient(config Config) (*gophercloud.ServiceClient, error) {
+func getIdentityClient(config ImageConfig) (*gophercloud.ServiceClient, error) {
 	opt := gophercloud.AuthOptions{
```

Comment:
if you move into controller, do you still need to do this? controller already has an identity client - I don't think it needs 2.